### PR TITLE
gmcksolid bigwig file

### PR DESCRIPTION
**Pull request by keyvanelhami**
Check list if changes affect bed files:

- [ ] Update shasum
- [ ] Update annotations
- [ ] Update filename if relevant
- [ ] Update person who designed/changed it
- [ ] Update bed file version
- [ ] Update UCSC track
- [ ] Update statusDB
- [ ] Deploy on Hasta
- [ ] Update LIMS

### CODE REVIEW

**Comment by keyvanelhami** [comment]
I have added a bigwig file for the gmcksolid panel for test.

**Comment by keyvanelhami** [comment]
The .md file is now added. Please come back if something else is needed

File: ucsc_track_hub/hg19/docs/BigWig_gmcksolid.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+Sample: NA24143
+
+Lims-id: ACC5812A1
+
+Lab Method: Twist
+
+Bioinformatic method: Analyzed with Balsamic 3.0.0 and .bw-file created with mosdepth
+
+### How the file was generated:
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmcksolid.md:28]:
```suggestion
## How the file was generated:
```

File: ucsc_track_hub/hg19/docs/BigWig_gmcksolid.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+1) Copy the above bedgraph file and remove the header using `vi`
+
+2) sort the copied file: `cat <cp.bedgraph> | sort -k1,1 -k2,2n > <cp.sorted.bedgraph>`
+
+3) Run: `bedGraphToBigWig <cp.sorted.bedgraph> https://hgdownload.cse.ucsc.edu/goldenPathhg19/bigZips/hg19.chrom.sizes <output.bw>`. The URL is a chrom.sizes file.
+
+
+
+
+## External links
```
Comment (hassanfa) [inline ucsc_track_hub/hg19/docs/BigWig_gmcksolid.md:61]:
```suggestion
# External links
```

**Pull request merged**